### ISSUE TITLE
Train on Pascal VOC

### ISSUE THREAD

**Issue by chenrxi** [facebookresearch/Mask2Former#122]
Is it possible to train it on dataset VOC

**Comment by bowenc0221** [comment]
Yes, you will need to register the VOC dataset following: https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html

**Comment by Yingyue-L** [comment]
For the semantic segmentation task, should I include the `background` category when using the VOC dataset?
